catalogue/webapp/pages/work.js

```diff
@@ -147,217 +137,13 @@ export const WorkPage = ({
-                      ])}>
-                      Download
-                      </h2>
-
-                      <div className={spacing({s: 2}, {margin: ['bottom']})}>
-                        <Button
-                          type='tertiary'
-                          url={convertImageUri(iiifImageLocationUrl, 'full')}
-                          target='_blank'
-                          download={`${work.id}.jpg`}
-                          rel='noopener noreferrer'
-                          trackingEvent={{
-                            category: 'Button',
-                            action: 'download large work image',
-                            label: work.id
-                          }}
-                          icon='download'
-                          text='Download full size' />
-                      </div>
-                      <div className={spacing({s: 3}, {margin: ['bottom']})}>
-                        <Button
-                          type='tertiary'
-                          url={convertImageUri(iiifImageLocationUrl, 760)}
-                          target='_blank'
-                          download={`${work.id}.jpg`}
-                          rel='noopener noreferrer'
-                          trackingEvent={{
-                            category: 'Button',
-                            action: 'download small work image',
-                            label: work.id
-                          }}
-                          icon='download'
-                          text='Download small (760px)' />
-                      </div>
-                    </Fragment>
-                  }
-
-                  {(iiifImageLocationCredit ||  iiifImageLocationLicenseId) &&
-                    <div className={spacing({s: 4}, {margin: ['bottom']})}>
-                      {iiifImageLocationCredit && <p className={classNames([
-                        font({s: 'HNL5', m: 'HNL4'}),
-                        spacing({s: 1}, {margin: ['bottom']})
-                      ])}>Credit: {iiifImageLocationCredit}</p>}
-                      {iiifImageLocationLicenseId && <License subject={''} licenseType={iiifImageLocationLicenseId} /> }
-                    </div>}
-
-                  <div className={spacing({s: 2}, {margin: ['top']})}>
-                    <Divider extraClasses={`divider--pumice divider--keyline ${spacing({s: 1}, {margin: ['top', 'bottom']})}`} />
-                    <h2 className={classNames([
-                      font({s: 'HNM4', m: 'HNM3'}),
-                      spacing({s: 2}, {margin: ['top']}),
-                      spacing({s: 1}, {margin: ['bottom']})
-                    ])}>
-                      Share
-                    </h2>
-                    <CopyUrl id={work.id} url={`https://wellcomecollection.org/works/${work.id}`} />
-                  </div>
-                </div>
-              </div>
-            </div>
-          </div>
-        }
+        <SingleColumnWork
```

Comment:
Does something along the path of `WorkDetails` feel more descriptive?

Reply:
Yep, I agree. I've branched off this to get the search box into an A/B test, so I'll merge this and update it in that PR.